### ISSUE TITLE
Learning rate schedule

### ISSUE THREAD

**Issue by doublejtoh** [yuhuixu1993/PC-DARTS#54]
Hello,
Regarding the initial learning rate of search process, 
paper noted 0.1 as the initial learning rate but actually code [here](https://github.com/yuhuixu1993/PC-DARTS/blob/master/V100_python1.0/train.py#L106) makes the initial learning rate 0.001 since lr scheduler is firstly executed.
I wonder if it is intended on purpose.
Thanks in advance.

Environments:
python3.*
pytorch1.4
cuda10.1
